File: client/src/app/shared/connection-details.ts
export class ConnectionDetails {
  constructor(con_type:string, device: string, name: string, uuid: string) {
    this.con_type = con_type;
    this.device = device;
    this.name = name;
    this.uuid = uuid;
  }
  con_type!: string;
  device!: string;
  name!: string;
  uuid!: string;
}

File: wifi-setup-service/connection_details.py
class ConnectionDetails:

    def __init__(self, name: str, uuid: str, c_type: str, device: str):
        self.name = name
        self.uuid = uuid
        self.con_type = c_type
        self.device = device

    name: str
    uuid: str
    con_type: str
    device: str
